client/src/components/VersionSaveComponent.tsx

```diff
@@ -0,0 +1,60 @@
+import { Button, Input } from 'antd'
+import React, { useState } from 'react'
+import { GetAnswerDocument, useSaveVersionMutation } from '../generated/graphql'
+import { messageService } from '../services/message'
+
+const VersionSaveComponent: React.FC<{
+  answerId: string
+  commitMessage: string
+  versionId: string
+}> = props => {
+  const [inputState, setInputState] = useState('')
+
+  const [saveVersion] = useSaveVersionMutation({
+    onCompleted: ({ saveVersion }) => {
+      if (saveVersion === true) {
```

Comment:
`saveVersion` already exists in the upper scope

```suggestion
    onCompleted: ({ saveVersion: didSaveVersion }) => {
      if (didSaveVersion) {
```